core/controller/src/main/scala/org/apache/openwhisk/core/loadBalancer/ShardingContainerPoolBalancer.scala

```diff
@@ -289,9 +289,13 @@ class ShardingContainerPoolBalancer(
 
     chosen
       .map { invoker =>
+        val memoryLimit = action.limits.memory
+        val memoryLimitInfo = if (memoryLimit == MemoryLimit()) { "std" } else { "non-std" }
```

Comment:
i'd create singletons for MemoryLimit and TimeLimit so that you're not consing new instances on every check.

Reply:
Thanks for your feedback. Hopefully, I understood it right. I will provide a change.